packages/@glimmer/util/lib/object-utils.ts

```diff
@@ -31,19 +31,11 @@ function assignFn(obj: any) {
 export let assign = Object.assign ?? assignFn;
 
 export function fillNulls<T>(count: number): T[] {
```

Comment:
I'd rather just remove this helper. It seems worse than just using `new Array(999).fill(null)` in each location.